rabbitsmpp/consumer.go

```diff
@@ -80,6 +90,17 @@ func (c *consumer) Consume() (<-chan Job, <-chan error, error) {
 		false,  // no-wait
 		nil,    // args
 	)
+}
+
+func (c *consumer) Consume() (<-chan Job, <-chan error, error) {
+	if c.channel != nil {
+		return nil, nil, errors.New("consumer already active")
+	}
+	closeChan, err := c.Bind()
```

Comment:
Since we want the first bind to succeed to complete the initialization, otherwise we would be blocking by retrying bind.